### PR TITLE
Update ru.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
added https://europaplustv.com/playlist

### CODE REVIEW

File: channels/ru.m3u [hunk 1]
```diff
@@ -60,6 +60,10 @@ http://ott-cdn.ucom.am/s78/index.m3u8
 https://sc.id-tv.kz/Cinema_34_35.m3u8
 #EXTINF:-1 tvg-id="Eurodance90HD.ru" tvg-name="Eurodance 90 HD" tvg-country="INT" tvg-language="English" tvg-logo="" group-title="Music",Eurodance 90 HD
 https://eurodance.catcast.tv/content/36987/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="RU" tvg-language="" tvg-logo="http://europaplustv.com/images/europa_tv.png" group-title="Music",EuropaPlusTV (1080p)
+https://ad-hls-europaplus.cdnvideo.ru/europaplus/smil:eurptv2.smil/playlist.m3u8?md5=-cus7xIweV03XfeaPG7xxw&e=1623093450&zoid=web
```
Comment (freearhey) [inline channels/ru.m3u:64]: image

File: channels/ru.m3u [hunk 2]
```diff
@@ -60,6 +60,10 @@ http://ott-cdn.ucom.am/s78/index.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="RU" tvg-language="" tvg-logo="http://europaplustv.com/images/europa_tv.png" group-title="Music",EuropaPlusTV (1080p)
+https://ad-hls-europaplus.cdnvideo.ru/europaplus/smil:eurptv2.smil/playlist.m3u8?md5=0N3nCDoPjwV57hi_EAs4YQ&e=1623149184&zoid=web
```
Comment (freearhey) [inline channels/ru.m3u:66]: image

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by Dum4G** [comment]
I think I'm gonna revert ClipAZ removal since discussion went nowhere. It is a constant source for the channel. Also persik.by links usually last long, even in public playlists http://persik.by/stream/6696/480248/682.m3u8

**Comment by LaneSh4d0w** [comment]
It's a generated link, I suspect https://loginza.ru/js/widget.js to be behind the "md5" generation.

**Comment by Sphinxroot** [comment]
is there anyway to make it work for other  ive been testing it for a week and still working https://i.imgur.com/L6aHv1N.png

**Comment by freearhey** [comment]
@Sphinxroot I just checked the result from the server and this is what I see:
image
It's probably just geo-blocked.

**Comment by LaneSh4d0w** [comment]
> 
> 
> @Sphinxroot I just checked the result from the server and this is what I see:
> image
> It's probably just geo-blocked.

I'm in the same country as him I presume, and I also have got a 403 Forbidden. Still watchable on website.

**Pull request closed without merging**